### ISSUE TITLE
[Feature Request] Test and develop Swate for Excel 2019 upwards.

### ISSUE THREAD

**Issue by Freymaurer** [nfdi4plants/Swate#177]
**Is your feature request related to a problem? Please describe.**
Office.js has several api sets, not all are accessible with all excel versions. The lowest version possible is excel 2016 with such restricted api options that it is not possible to even use base Swate functionality. While a lot of universities do not provide staff/students with office 356 versions and altough Swate will always work in the webversion it might be useful to check for versions 2019/2021.

**Describe the solution you'd like**
Need a VM with the above mentioned excel version for development and testing.

**Comment by Freymaurer** [comment]
Got the first minimal exmaple for excel 2019 working, so finally some progress.

**Comment by elisenger** [comment]
Hi,
I am using Excel 2019 on Win10. Swate is installed and when I click on the small arrow to add "my add-ins" I can select it and it shows up in the data menu as buttons "core" and "experts", but when I click these buttons, just empty side bar pops up. Any clue what is the problem? thanks :)

**Comment by Freymaurer** [comment]
Yes indeed! it took me the last few weeks to narrow down the problem. 

Excel 2019 still uses Internet Explorer 11 to open the add-ins, which does not support modern java script. Due to the interactions of the framworks on which Swate is based and on their newest updates, the back compilation to a javascript version functionable with IE 11 was removed. Now that we found the problem i got a minimal version running in 2019 and now i can update Swate accordingly!

tl;dr: Updates in Swate dependencies removed compatibility with IE 11 and therefore Excel 2019. Problem found, solution halfway through, working on it with highest priority!

**Comment by Freymaurer** [comment]
With the newest release Swate 0.5.1 we now support Excel 2019 on both windows and mac. Feel free to give it a try and report any errors which may occur to us!
